docs/resources/kafka_connect_cluster_v2.md

```diff
@@ -0,0 +1,320 @@
+- **_optional_** - These input attributes can be omitted, and doing so may result in a default value being used.
+- **_immutable_** - These are input attributes that cannot be changed after the resource is created.
+- **_updatable_** - These input attributes can be updated to a different value if needed, and doing so will trigger an update operation.
+- **_nested block_** - These attributes use the [Terraform block syntax](https://www.terraform.io/language/attr-as-blocks) when defined as an input in the Terraform code. Attributes with the type **_repeatable nested block_** are the same except that the nested block can be defined multiple times with varying nested attributes. When reading nested block attributes, an index must be provided when accessing the contents of the nested block, example - `my_resource.nested_block_attribute[0].nested_attribute`.
+## Root Level Schema
+### Input attributes - Required
+*___data_centre___*<br>
+<ins>Type</ins>: nested block, required, updatable, see [data_centre](#nested--data_centre) for nested schema<br>
+<ins>Constraints</ins>: minimum items: 1<br><br>List of data centre settings.<br><br>
+*___sla_tier___*<br>
+<ins>Type</ins>: string, required, immutable<br>
+<ins>Constraints</ins>: allowed values: [ `PRODUCTION`, `NON_PRODUCTION` ]<br><br>SLA Tier of the cluster. Non-production clusters may receive lower priority support and reduced SLAs. Production tier is not available when using Developer class nodes. See [SLA Tier](https://www.instaclustr.com/support/documentation/useful-information/sla-tier/) for more information.<br><br>
+*___kafka_connect_version___*<br>
+<ins>Type</ins>: string, required, immutable<br>
+<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Version of Kafka connect to run on the cluster. Available versions: <ul> <li>`3.1.2`</li> <li>`3.0.2`</li> <li>`2.8.2`</li> <li>`2.7.1`</li> </ul><br><br>
+*___name___*<br>
+<ins>Type</ins>: string, required, immutable<br>
+<ins>Constraints</ins>: pattern: `[a-zA-Z0-9][a-zA-Z0-9_-]*`<br><br>Name of the cluster.<br><br>
+*___private_network_cluster___*<br>
+<ins>Type</ins>: boolean, required, immutable<br>
+<br>Creates the cluster with private network only, see [Private Network Clusters](https://www.instaclustr.com/support/documentation/useful-information/private-network-clusters/).<br><br>
+*___target_cluster___*<br>
+<ins>Type</ins>: nested block, required, immutable, see [target_cluster](#nested--target_cluster) for nested schema<br>
+<br>Details to connect to a target Kafka Cluster cluster.<br><br>
+### Input attributes - Optional
+*___custom_connectors___*<br>
+<ins>Type</ins>: nested block, optional, immutable, see [custom_connectors](#nested--custom_connectors) for nested schema<br>
+<br>Defines the location for custom connector storage and access info.<br><br>
+*___two_factor_delete___*<br>
+<ins>Type</ins>: nested block, optional, updatable, see [two_factor_delete](#nested--two_factor_delete) for nested schema<br>
```

Comment:
Do we have an updatable feature of this in the app repo for Kafka Connect?

Reply:
Yes, Number of nodes, which comes under dataCentre. This is the same as the APIV2 signature

Reply:
I mean this one ___two_factor_delete___, which is defined as updatable.